docker/Dockerfile

```diff
@@ -16,5 +16,9 @@ RUN set -ex; \
 
 RUN set -ex; \
     pip3 --no-cache-dir install --no-deps ldap2pg; \
```

Comment:
This `RUN` instruction smells like it would stay cached between `ldap2pg` releases. In my experience, this is usually handled by including the desired version as an `ARG` or directly in the Dockerfile. For example,

https://github.com/docker-library/postgres/blob/cc305ee1c59d93ac1808108edda6556b879374a4/11/Dockerfile#L74

Reply:
Hmm, good point. I didn't encountered such issue yet. I have enabled build-caching in docker automated build. I'm lazy to add an extra step in release process to keep Dockerfile sync. I suggest to wait and see what happen on next release. If build on push don't update versions, I'll apply your trick. What do you think of this plan ?

Reply:
Fine by me!

Reply:
Thanks Chris :-)